active/ZEP0000.md

```diff
@@ -130,16 +130,17 @@ mentioned below. This allows the author to flesh out the draft ZEP to make it
 properly formatted, of high quality, and address initial concerns about the
 proposal.
 
-After the PR for the ZEP is in place, a post should be made on the GitHub
-organisation [discussions], containing the sections up to “Backward
-compatibility” to limit discussion there to usage and impact. Discussion on the
-pull request will have a broader scope, including details of implementation.
+After the PR for the ZEP is in place, an issue should be created in the [ZEPs]
```

Comment:
Sorry, why a PR and an issue? Is there an equivalent issue today e.g. for ZEP1 that I could look at as an example of what you mean?

Reply:
The PR is created in the [ZEPs](https://github.com/zarr-developers/zeps/) repository under [draft](https://github.com/zarr-developers/zeps/tree/main/draft) folder, which contains the narrative document. Once merged, it will be rendered under [draft_zeps](https://zarr.dev/zeps/draft_zeps/). The issue is supposed to hold the discussion on the ZEP after PR is merged because having a discussion on merged PR is not ideal.

> Is there an equivalent issue today e.g. for ZEP1 that I could look at as an example of what you mean?

Not for ZEP0001, but something similar was created for ZEP0003. Check [here](https://github.com/orgs/zarr-developers/discussions/52).

```
an issue should be created in the ZEPs containing the sections up to “Backward compatibility” to limit
discussion there to usage and impact. Discussion on the pull request will have
a broader scope, including details of implementation.
```

I think we can amend the above statement to something like...

```
an issue should be created in ZEPs to hold discuss the various sections of the ZEP like backward compatibility, usage, impact, scope and details of implementation.
```

to make it more clear. After this we'll have one PR to `zeps/draft` and one issue in the `zeps` repository to hold all the comments and discussion.

Let me know what you think. Thanks!

Reply:
:+1: for trying to clarify this (if even I'm confused 🙂 )

Reply:
Resolved this in the latest commit.

The discussion related to backward compatibility, usage, impact etc., can occur in PR in the `zarr-specs` repo.

Something like https://github.com/zarr-developers/zarr-specs/pull/152 and https://github.com/zarr-developers/zarr-specs/pull/205.

If an author still needs to create a separate issue for discussing the above points, they can create an issue, and I have stated that in the `Discussing a ZEP` section.